transports/azure-storage-queues/multi-storageaccount-support_aliases_asq_[7,].partial.md

```diff
@@ -1,25 +1,14 @@
-## Using aliases for connection strings to storage accounts for Scale Out
```

Comment:
The filename is off.  Shouldn't the range be `[7,)`? I.e. from version 7 (included) and up.

Reply:
@SeanFeldman I think that if you don't specify the upper version limit number then it works the same way. So you can use either ] or ).